### DISCUSSION TITLE
portifólio frontend no github

### DISCUSSION THREAD

**Discussion by RicardoBaltazar** [frontendbr/forum#1797] [Discussão]
Olá, vocês tem algumas dicas de como posso usar melhor o github como portifólio de projetos frontend?

**Comment by romulo2735** [comment]
Uma duvida que tenho também...

**Comment by felipefialho** [comment]
Oi gente, qual a dúvida exatamente?

O Github é uma rede social como qualquer outra, tudo que vocês fazem aqui pode ser visto como "parte do seu portfólio".  Escrevi esse texto uns anos atrás, talvez ajude na ideia:

- [Como contribuir com open source](https://www.felipefialho.com/blog/como-contribuir-com-open-source/)

**Comment by kvnol** [comment]
Concordo total com o @felipefialho e é até normal recebermos e-mails com propostas de RHs que viram nossos perfis, por isso é sempre legal deixar tudo organizado e atualizado.

Também escrevi um artigo um tempo atrás sobre contribuição no GitHub, que é o que atrai pessoas ao seu perfil:

- [Como contribuir em projetos open source sem codar](https://kevinoliveira.com.br/blog/como-contribuir-em-projetos-open-source/)

**Comment by romulo2735** [comment]
No meu caso precisa saber como montar portifolio para mostrar para os ponteciais clientes e empresas, por exemplo, não precisa o usuário instalar e clonar o projeto, mas apenas ele tem acesso o link com imagens que como ficou o projeto, mais ou menos isso, Pelo menos para mim.

**Comment by gabriel-hahn** [comment]
Por exemplo, tu pode utilizar o Github pages, Netlify ou o Heroku para manter o site online e tu enviar o link aos clientes. Outra alternativa é criar GIFs e imagens do teu projeto e colocar no README do mesmo, dando uma visão do projeto sem precisar que o cliente rode ele localmente.
A melhor opção seria tu ter um site que centralizasse toda essa informação, por exemplo criar um site de portfólio no qual o cliente teria acesso a todos os teus projetos listados e conseguisse ler sobre eles e acessar. Github não é muito amigável para pessoas não técnicas, se esse for o perfil que tu quer atingir :/

**Comment by romulo2735** [comment]
@gabriel-hahn Pronto quero isso, vi por cima o github pages, seria o nome do projeto, descrição e fotos, videos ou gif para ver o projeto rodando e tals. Vou procurar eles, no github pages uso markdown para fazer?

**Comment by ninetails** [comment]
@romulo2735 é possível qualquer site estático.

> https://pages.github.com/

O GitHub pages tem [Jekyll](https://jekyllrb.com/docs/github-pages/) integrado, mas não sei como funciona pois nunca utilizei.
